example.py

```diff
@@ -21,12 +21,12 @@ async def main():
     await device.clear_screen("ffffffff")
 
     text = "Hello, world!"
-    font = fonts.GYWFonts.LARGE
-    text_drawing = drawings.TextDrawing(text=text, left=100, top=100, font=font, color="ff000000")
+    font = fonts.GYWFonts.ROBOTO_MONO
+    text_drawing = drawings.TextDrawing(text=text, left=100, top=100, font=font, size=24, color="ff000000")
     await device.send_drawing(text_drawing)
 
     text = "Big green text!"
-    text_drawing = drawings.TextDrawing(text=text, left=100, top=350, size=80, color="ff00ff00")
+    text_drawing = drawings.TextDrawing(text=text, left=100, top=350, font=font, size=80, color="ff00ff00")
```

Comment:
In the example, I especially used a font that is not the default one at the top (`LARGE` instead of `MEDIUM`), and then I voluntarily not set the font so that it uses the default font. Can you do something similar here ?

Reply:
Right. I forgot to change it back when I reintroduced default values.